frontend/views/containers/navigation/Navigation.vue

```diff
@@ -187,14 +186,12 @@ export default ({
     openModal (mode) {
       sbp('okTurtles.events/emit', OPEN_MODAL, mode)
     },
-    enableTimeTravel (evt) {
-      if (evt.shiftKey && process.env.NODE_ENV !== 'production') {
-        console.debug('enable time travel!')
-        this.ephemeral.timeTravelComponentName = 'TimeTravel'
-      }
-    },
```

Comment:
I noticed this logic related to time-travel is no longer used anywhere in the app. So just removed it.